src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -24,6 +24,11 @@ extern "C" {
  */
 #define MAX_CANDIDATE_ID_LENGTH 9U
 
+/**
+ * Maximum allowed ICE URI length
+ */
+#define MAX_ICE_CONFIG_URI_LEN 127
```

Comment:
Looks fine.

```
turns:xx-xx-192-130.t-xxxxxxxx.kinesisvideo.us-west-2.amazonaws.com:443?transport=tcp
```

This URL is 85 chars long and other regions with special prefix and suffixes, we have about 40 character buffer for additional stuff.